arwen/contexts/managedType.go

```diff
@@ -56,6 +58,19 @@ func NewManagedTypesContext(host arwen.VMHost) (*managedTypesContext, error) {
 	return context, nil
 }
 
+func initRandomizer(host arwen.VMHost) *math.SeedRandReader {
```

Comment:
+ add a unit test. Same here: hardcoded values that will always produce the same outcome. Backwards compatibility and deterministic results are important.